lib/Language/en/translation.json

```diff
@@ -789,6 +789,16 @@
       "csvRegionMappingMessageLoadError": "Couldn't load region boundaries for region {{regionName}} {{exception}}"
     }
   },
+  "cesiumExtraCredits": {
```

Comment:
hmm, i don't think the actual link should be in the translation.json as this won't change if its a different language. Keep in mind that external translators contribute to this file as well through weblate

Reply:
i reckon keep the link hardcoded if it doesn't have to change now but if it does, perhaps a better place would be inside config.json

Reply:
Yeah, it was sort of a quick fix. I was holding back from adding more new config parameters. I guess I'll redo the PR adding config parameters and making it less hackier.